protocol/tbdex_schema_examples.js

```diff
@@ -0,0 +1,50 @@
+let offering = {
+  "pair": "BTC_USD",
+  "unitPrice": 27000.00,
+  "fee": 1.00,
+  "min": 10.00,
+  "max": 100.00,
+  "presentationRequest": { },
+  "payinInstruments": [{
+      "kind": "DEBIT_CARD",
+      "fee": 1,
+      "presentationRequest": {},
+  }],
+  "payoutInstruments": [{
+      "kind": "BTC_ADDRESS",
+      "presentationRequest": {}
+  }]
+}
+
+let rfq = {
+  "rfqId": "1",
+  "pair": "BTC_USD",
+  "amount": 10.00,
+  "verifiablePresentation": { },
+  "payin_instrument": {
+      "kind": "DEBIT_CARD"
+  },
+  "payout_instrument": {
+      "kind": "BTC_ADDRESS"
+  }
+}
+
+let quote = {
+  "rfqId": "1",
+  "quoteId": "2",
+  "expiryTime": "2023-04-14T12:12:12Z",
+  "totalFee": 1.00,
+  "amount": 0.000383,
+  "paymentPresentationRequest": { }
```

Comment:
includes both payin and payout VCs in 1 presentation request

Reply:
maybe illustrate that in the actual object here as an example

Reply:
done! presentationRequest would actually be in the format of a JWT strings, so i changed it and added a "realistic" presentation request jwt which you can decode using jwt.io to peek at the presentation definition inside.